modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -12467,6 +12468,7 @@ no-information-was-found-associated-with-the-symbol-x=No information was found a
 no-instances-were-found=No instances were found.
 no-issues-found-in-this-component=No issues found in this component.
 no-items=No Items
+no-items-shared-with-you-yet=No Items Shared with You Yet
```

Comment:
I'm not sure this is right, I see in the figma all of the words are capitalized except with as you have duplicated here, but to me it does not make sense, do you know the reason behind this choice? I feel like it should either all be capitalized, or only the first word as it is not a title.